lib/elixir/test/elixir/integer_test.exs

```diff
@@ -3,9 +3,9 @@ Code.require_file "test_helper.exs", __DIR__
 defmodule IntegerTest do
   use ExUnit.Case, async: true
 
+  require Integer
   doctest Integer
 
-  require Integer
```

Comment:
Can probably revert this now, right? Not totally necessary, but minimizes diff noise. :)